charts/caraml/values.yaml

```diff
@@ -1,3 +1,43 @@
 # Default values for caraml.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
+
+mlp:
+  enabled: true
+  postgresql:
+    enabled: true
```

Comment:
by default, we should have one postgresql instance. hence `mlp.postgresql.enabled` should be false and `mlp.enabled` true

Reply:
changed